src/test/java/core/basesyntax/service/FileReaderServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static final String FILE_HEADER = "type,fruit,quantity";
+    private static final String testTransactionFilePath = "src/main/resources/transaction.csv";
```

Comment:
What about second constnat? If mentor writes some comment, it means that it should be fixed in all such places